### PR TITLE
fix(cgi): Change key to re-order list of disks

### PR DESCRIPTION
**Pull request by Antuan-AE**
- A buggy implementation of sort algorithm was placing last entry as first one.

### CODE REVIEW

**Review by rolysr — APPROVED** [review]
LGTM

File: src/cgi/sfs.cgi.in [hunk 1]
```diff
@@ -1652,7 +1652,8 @@ if "HD" in sectionset:
                             wtime = usecwritemax
                             fsynctime = usecfsyncmax
                         if HDorder == 1 or HDorder == 0:
-                            sf = (ip1, ip2, ip3, ip4, port, data[1:plen + 1])
+                            sf = (ip1, ip2, ip3, ip4, port, entry[1:plen + 1]
+                                  .decode('utf-8'))
```
Comment (rolysr) [inline src/cgi/sfs.cgi.in:1656]: Is there a test checking for this to be correctly working?

**Review by rolysr — APPROVED** [review]

File: src/cgi/sfs.cgi.in (hunk 1 above)
Comment (Antuan-AE) [reply src/cgi/sfs.cgi.in:1656]: Even though there is a test for entire cgi, this issue is not reported and was accidentally detected.

Before modification:
image

After modification:
image

Comment (rolysr) [reply src/cgi/sfs.cgi.in:1656]: Ok perfect

**Review by lgsilva3087 — APPROVED** [review]
LGTM.

**Pull request merged**